kinto_http/session.py

```diff
@@ -71,8 +83,10 @@ def request(self, method, endpoint, data=None, permissions=None,
             payload_kwarg = 'data' if 'files' in kwargs else 'json'
             kwargs.setdefault(payload_kwarg, payload)
 
+
         retry = self.nb_retry
         while retry >= 0:
+            kwargs['headers']={'User-Agent':USER_AGENT}
```

Comment:
Here I suggest that we don't override existing values in `kwargs`. So please do this instead:
```py
kwargs.setdefault('headers', {}).setdefault('User-Agent', USER_AGENT)
```